frontend/src/hooks/useUsers.ts

```diff
@@ -0,0 +1,15 @@
+import useSWR from 'swr';
+
+import { userAPI } from '../config/usersAPI';
+import type { User } from '../config/user.types';
+
+export const useUsers = () => {
+  const { data, error, mutate } = useSWR<User[]>('users', userAPI.getAll);
```

Comment:
SWR provides `isLoading` which is more accurate and true only for initial load
```suggestion
  const { data, error, isLoading, isValidating, mutate } = useSWR<User[]>('users', userAPI.getAll);
```